.eslintrc.js

```diff
@@ -103,6 +111,7 @@ module.exports = {
         },
       },
     ],
+    '@preconstruct/format-js-tag/format': 'error',
```

Comment:
This ESLint rule formats things like this with prettier so we can write JS code in JS that we need to write to a file without having to format it manually. (it's not used in the test in this PR but will be used in the future for tests around compilation)

```js
js`
   console.log(1);
`
```